cli/scripts/monorepo.ts

```diff
@@ -323,15 +326,7 @@ fs.copyFileSync(
     args?: readonly string[],
     options?: execa.SyncOptions<string>
   ) {
-    const resolvedArgs = [...args];
-
-    // Include these to make sure we don't error.
-    if (command == "run") {
-      resolvedArgs.unshift("--experimental-use-daemon");
-      resolvedArgs.unshift("--stream");
-    }
```

Comment:
These are deprecated and we're pulling the trigger on removing them instead of porting them, correct?

Reply:
The problem is that those are going to exist in CI setups. The test there is simply to make sure we don't go boom when we have them sent in.

So: they're fine to do literally nothing, but this test itself should stay so that we don't accidentally exit 1.